composer.json

```diff
@@ -11,7 +11,6 @@
     }
   ],
   "minimum-stability": "stable",
-  "version": "1.1.0",
```

Comment:
`composer.json` strict validation fails. Apparently composer is so well integrated with git, they only want to use git tags for versioning, not this file 🤷‍♂️ 

See this blog post for more info: https://medium.com/packagist/tagged-a-new-release-for-composer-and-it-wont-show-up-on-packagist-org-or-on-private-packagist-efaf21c212ff